### ISSUE TITLE
How Install HyperionNG  in CoreELEC or Libreelec 

### ISSUE THREAD

**Issue by LORDBLADE2000** [hyperion-project/hyperion.ng#555]
Hi Friends

I Have Aml box ,witih Corelec  ,i want install Hyperion NG,but I did not understand wiki .

**Comment by LuckyTNT** [comment]
Hi, I would like to extend this to OSMC, when I try to run "Compiling and installing Hyperion" part from "CompileHowto":
wget -qO- https://raw.githubusercontent.com/hyperion-project/hyperion.ng/master/bin/compile.sh | sh
I get error:
https://gist.github.com/LuckyTNT/668e1a341a1c125cd9e77cfcfbc0d353
I am not sure where to go next as error is not self-explaining enough for me.

I done it fine a year or so ago with some older version 1.8 or 1.9 if i remember correctly, didn't have problem then. I set it and forget it then. Had only to install some dependencies. 

I tryed to compile on docker with Hyperion for Raspberry Pi 2 & 3 as well and compiling finished but when I try to install deb package it opens new problems.

I'll be glad to do it on other sd card again and write full step by step guide for others on OSMC if someone get me trough it once. :)
